read_mars/mstatusdisplay_to_dict.py

```diff
@@ -0,0 +1,143 @@
+from collections import OrderedDict
+import numpy as np
+import ROOT
+
+import fact
+
+
+def mstatusdisplay_to_dict(
+        path,
+        transform=True,
+        exclude=['TF1', 'TPaveText'],
+        ):
+    ''' return a flat dict, with string keys and values either coming:
+     * from an MHCamera -> np.array(shape=(1440,)) or
+     * from a TH1 -> {'bins': np.array, 'content':np.array}
+
+    keep low  level acces to the Root classes with `transform=False`
+    and more class names to the `classes` list in case you want ot
+    '''
+
+    tfile = ROOT.TFile(path)
+    status_array = tfile.Get('MStatusDisplay')
+    canvases = canvases_from_status_array(status_array)
+
+    contents = OrderedDict()
+    for name, canvas in canvases.items():
+        contents[name] = primitives_from_canvas(canvas)
+    contents = flatten_ordered_dicts(contents)
+
+    if transform:
+        output = {}
+        for name, obj in contents.items():
+            clsname = obj.__class__.__name__
+            if clsname in exclude:
+                continue
+            if clsname == 'MHCamera':
+                output[name] = transform_MHCamera(obj)
+            elif clsname in ['TH1F', 'TH1D']:
+                output[name] = transform_TH1(obj)
+            else:
+                output[name] = obj
+        return output
+    else:
+        return contents
+
+
+def transform_TH1(h):
```

Comment:
numpy also has bins, but there it is every edge, not just all lower edges. Maybe do the same here?